src/ImageUtility.ps1

```diff
@@ -0,0 +1,38 @@
+﻿function Export-Entity([string] $entityPath, [string[]] $fileNames)
+{
+    $srcDir = [io.path]::combine($Config.AuditDir, $entityPath)
+    $dstDir = [io.path]::combine($Config.ArtifactsDir, $entityPath)
+
+    New-Item -Path $dstDir -ItemType directory -Force
+
+    Get-ChildItem -path $srcDir | ForEach-Object {

```

Comment:
Maybe this will help:
```powershell
ls $srcDir -Filter '*.jpg' -Recurse | % {
    $postfix = if ($_.Name -like '*.small.*') { '.small' } else { '' }
    $dstFile = Join-Path $dstDir ("$($_.DirectoryName)$postfix.jpg")
    Copy-Item $_.FullName $dstFile -WhatIf 
}
```